### ISSUE TITLE
[1.14] Towny server lags 

### ISSUE THREAD

**Issue by Feldmarschall** [TownyAdvanced/Towny#3240]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. default 1.14 spigot server with towny 0.93.0.0
2. /timings on -> wait 4-6 hours
3. /timings paste -> https://imgur.com/a/xgXIEmW

Problem:
High performance lags, because townys "MobRemovalTimerTask".

Towny v0.93.0.0 Total: 10.9 s	Pct: 41.79%
Pct Total	Pct Tick	Total	Avg	PerTick	Count	Event
41.76%	4,356.52%	10.89 s	2,178.26 ms	0.0	0.0k	Task: MobRemovalTimerTask(interval 100)

https://imgur.com/a/xgXIEmW

### What is your Towny version?
0.93.0.0

### What is your spigot/craftbukkit version?

git-Spigot-1eece4f-a584320 - 1.14-R0.1-Snapshot

1. Your full server startup from the logs\latest.log : https://pastebin.com/LvDC6rWV
2. Your Towny config.yml (if using MySQL please remove password,) : default config
3. Your townyperms.yml : default settings 
4. Your log's error : no errors

**Comment by LlmDl** [comment]
Link to original ticket: https://github.com/TownyAdvanced/Towny/issues/3237
@BarnabyD

Can you paste the full timings link? Without it, it can be hard to tell how much of an impact Towny is causing.

**Comment by BarnabyD** [comment]
World#getLivingEntities does appear to only include mobs in loaded chunks.

```
List<LivingEntities> livingEntities = world.getLivingEntities();
System.out.println("Towny Debug: getLivingEntities Total " + livingEntities.size());

int isChunkLoadedTotal = 0;
for (LivingEntity livingEntity : livingEntities) {
   Location livingEntityLoc = livingEntity.getLocation();
   
   if (!livingEntityLoc.getChunk().isLoaded())
      continue;
   
   isChunkLoadedTotal++;

   ...
}

System.out.println("Towny Debug: getLivingEntities Chunk Loaded Total " + isChunkLoadedTotal);
```

livingEntities.size() and isChunkLoadedTotal was always the same number, no matter what I tried doing.

**Comment by BarnabyD** [comment]
Here are my own timings before and after the above change:
Before - [https://timings.spigotmc.org/?url=hipikuduxa](https://timings.spigotmc.org/?url=hipikuduxa)
After - [https://timings.spigotmc.org/?url=efifasubon](https://timings.spigotmc.org/?url=efifasubon)

**Comment by LlmDl** [comment]
Looks interesting. I think your PR will be accepted. Thanks.

**Comment by LlmDl** [comment]
Fixed as of 0.93.1.0, thanks for the PR.
